ResearchKit/Common/ORKStepViewController.m

```diff
@@ -345,6 +345,35 @@ - (void)showValidityAlertWithMessage:(NSString *)text {
     }];
 }
 
+- (void)showErrorAlertWithTitle:(NSString *)title message:(NSString *)message {
```

Comment:
Refactor this to be `showValidityAlertWIthTitle: message:` to stay consistent with the already available `showValidityAlertWithMessage`. And inside `showValidityAlertWithMessage` make a call to the new `showValidityAlertWithTitle: message:`.